pkg/click/README.md

```diff
@@ -0,0 +1,514 @@
+    click [-o|-p] -e -i <path-to-file> <path-to-pier>
+
+    Thin client for interacting with running Urbit ship via conn.c
+
+    options:
+        -e                  Execute jammed Hoon
+        -h                  Show usage info
+        -i <path-to-file>   Read input from file
+        -j                  Jam only
+        -k                  Execute command using "khan-eval" thread
+        -o <path-to-file>   Output to file
+        -p                  Filter failure stack traces from result and pretty-print them to stderr
+        -x                  Jam to hex
+```
+
+## Using these tools
+
+Below are examples of how to execute common commands on a running ship from
+Earth.
+
+### `|mass`
+
+Blocked by issues; not currently doable in a way that returns the results as
+data.
+
+### `|pack`
+
+```
+echo "[0 %urth %pack]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+echo "[0 %ovum %d /test %pack ~]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+/path/to/click -kp /path/to/pier/zod \
+$'=/  m  (strand ,vase)  ;<  ~  bind:m  (flog [%pack ~])  (pure:m !>(\\\'success\\\'))'
+```
+
+### `|meld`
+
+```
+echo "[0 %urth %meld]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+echo "[0 %ovum %d /test %meld ~]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+/path/to/click -kp /path/to/pier/zod \
+$'=/  m  (strand ,vase)  ;<  ~  bind:m  (flog [%meld ~])  (pure:m !>(\\\'success\\\'))'
+```
+
+### `|ota`
+
+#### `|ota ~bus`
+
+```
+echo "[0 %ovum [%g /test [%deal [~zod ~zod] %hood %raw-poke %kiln-install %base ~bus %kids]]]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+/path/to/click -kp /path/to/pier/zod \
+$'=/  m  (strand ,vase)  ;<  our=@p  bind:m  get-our  ;<  ~  bind:m  (poke [our %hood] %kiln-install !>([%base ~bus %kids]))  (pure:m !>(\\\'success\\\'))'
+```
+
+#### `|ota %disable`
+
+```
+echo "[0 %ovum [%g /test [%deal [~zod ~zod] %hood %raw-poke %kiln-install %base ~zod %base]]]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+/path/to/click -kp /path/to/pier/zod \
+$'=/  m  (strand ,vase)  ;<  our=@p  bind:m  get-our  ;<  ~  bind:m  (poke [our %hood] %kiln-install !>([%base our %base]))  (pure:m !>(\\\'success\\\'))'
+```
+
+#### `|ota ~bus %desk`
+
+```
+echo "[0 %ovum [%g /test [%deal [~zod ~zod] %hood %raw-poke %kiln-install %base ~zod %desk]]]" |
+/path/to/urbit eval -jn |
+nc -U -W 1 /path/to/pier/zod/.urb/conn.sock |
+/path/to/urbit eval -cn
+```
+```
+/path/to/click -kp /path/to/pier/zod \
+$'=/  m  (strand ,vase)  ;<  our=@p  bind:m  get-our  ;<  ~  bind:m  (poke [our %hood] %kiln-install !>([%base our %desk]))  (pure:m !>(\\\'success\\\'))'
```

Comment:
Should this be `[%base ~bus %desk]` instead of `[%base our %desk]`?

Reply:
You're correct - c-p typo from the above command